testing_dir/minesweeper.py

```diff
@@ -36,7 +36,7 @@ def print_stats():
 def update_player_pos(direction):
     x, y = player_pos
     if direction == "up":
-        y -= 1
+        y -= 2  # Error: should be 1
```

Comment:
The decrement value for moving "up" is incorrect. It should be consistent with other directions.
```diff
-        y -= 2  # Error: should be 1
+        y -= 1
```